File: patterns/c-patterns/pattern15.c
// Pattern 15: ...

// 12345
// 1234*
// 123**
// 12***
// 1****

#include <stdio.h>

int main()
{
    int i, j;
    int n = 5;
    for (i = n; i >= 1; i--)
    {
        for (j = 1; j <= n; j++)
        {
            if (j > i)
                printf("*");
            else
                printf("%d", j);
        }
        printf("\n");
    }
    return 0;
}

File: patterns/javascript-patterns/pattern15.js
// Pattern 15: ...

// 12345
// 1234*
// 123**
// 12***
// 1****

const n = 5;

for (let i = n; i >= 1; i--) {
    let row = '';

    for (let j = 1; j <= n; j++) {
        if (j > i) {
            row += '*';
        } else {
            row += j;
        }
    }

    console.log(row);
}
